packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -233,7 +233,7 @@ class _GlowingOverscrollIndicatorState extends State<GlowingOverscrollIndicator>
       final bool isLeading = controller == _leadingController;
       if (_lastNotificationType is! OverscrollNotification) {
         final OverscrollIndicatorNotification confirmationNotification = OverscrollIndicatorNotification(leading: isLeading);
-        confirmationNotification.dispatch(context);
+        context.dispatchNotification(confirmationNotification);
```

Comment:
here and elsewhere: revert this since it's just unnecessary churn now?

Reply:
done